### PR TITLE
readme edits

### LINKED ISSUE #7: Add Jillian team member info to backend README
Please open a PR which modifies the  frontend `README` to add info about yourself at the top:

At the very top of the doc, please add an h1 heading for `Team Members`

And then add your information as a bullet point:

- **myFirstName** (@myGithubUsername) someInfoAboutMe 

Formatting:
* add bold to your name
* add a link to your username pointing to your github profile page

You can see more info about Github's markdown syntax in context from the existing `README` content, as well as in the documentation [here](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)

### PR DESCRIPTION
**Pull request by catbus00**
Fixes #7

### CODE REVIEW

**Review by dargaCode — CHANGES_REQUESTED** [review]
Nice can you please make the same changes to this one as well?

File: package.json [hunk 1]
```diff
@@ -27,6 +27,6 @@
     "eslint-plugin-n": "^15.7.0",
     "eslint-plugin-promise": "^6.1.1",
     "mocha": "^10.2.0",
-    "nodemon": "^2.0.21"
+    "nodemon": "^2.0.22"
```
Comment (dargaCode) [inline package.json:30]: Do you know how this `nodemon` version got changed? Seems like something that should have only happened if you were maybe missing package-lock.json or told it to update somehow manually?

**Review by dargaCode — CHANGES_REQUESTED** [review]
Thanks for the revisions! I have one more suggestion for practice, and a question about this `nodemon` stuff which I think we might need to revert?

File: package.json (hunk 1 above)
Comment (catbus00) [reply package.json]: It must have automatically upgraded it when I was playing around with some features, so I changed it back to the original version.

Comment (dargaCode) [reply package.json]: Only PR #13 needs these script/package changes, so I'm going to remove them here. (please see [slack thread](https://codethedream.slack.com/archives/C060LAKRWTG/p1699122656551589))

**Comment by dargaCode** [comment]
Only PR #13 needs these script/package changes, so I'm going to remove them here. (please see [slack thread](https://codethedream.slack.com/archives/C060LAKRWTG/p1699122656551589))

File: README.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# Team Members
+
+* **Jillian** \([@catbus00](https://github.com/catbus00)\) is a student studying React, but showing Backend Team some support!
```
Comment (dargaCode) [inline README.md:3]: 🙌

**Review by dargaCode — APPROVED** [review]
Great work, thank you Jillian!

**Comment by dargaCode** [comment]
Normally you'd be the one to merge after a final review, but I'm doing it here so that I can be sure my changes didn't cause any problems

**Pull request merged**